Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml

```
@@ -0,0 +1,303 @@
+- type: entity
+  id: WeaponMechCombatPulseRifle
+  name: eZ-14 mk2 Heavy pulse rifle
+  description: Fires a heavy pulse laser.
+  suffix: Mech Weapon, Gun, Combat, Pulse
+  parent: [ BaseMechWeaponRange, CombatMechEquipment ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)